src/core/fonts.js

```diff
@@ -3216,10 +3216,14 @@ var Font = (function FontClosure() {
         };
       }
 
-      var fontChar =
-        typeof fontCharCode === "number"
-          ? String.fromCodePoint(fontCharCode)
-          : "";
+      let fontChar = "";
+      if (typeof fontCharCode === "number") {
+        if (fontCharCode < 0x110000) {
```

Comment:
Would it perhaps be better to use `< 0 or > 0x10FFFF` just like in the specification, so also negative numbers are handled here?

Reply:
> Would it perhaps be better to check for `< 0 or > 0x10FFFF` just like in the specification,

I thought about the `< 0` case, but honestly it seems to me that that would indicate *much larger* underlying problems in the code to the point that I'd not mind a breaking error (also considering that the `String.fromCodePoint` changes landed ~1.5 years ago and this is the first time they've caused a problem).

> can we at least change this to `<= 0x10FFFF` so it's easier to match with the spec?

If you think the current formatting is too confusing, maybe because of the amount of digits involved, I suppose that that can be changed :-)